mysql/mysqldump.go

```diff
@@ -321,6 +360,11 @@ func processAlterTable(conv *internal.Conv, stmt *ast.AlterTableStmt) {
 					ctable.ForeignKeys = append(ctable.ForeignKeys, constraint.fk)
 					conv.SrcSchema[tableName] = ctable
 				}
+				if constraint.isUniqueKey {
+					ctable := conv.SrcSchema[tableName]
+					ctable.Indexes = append(ctable.Indexes, schema.Index{Name: "", Unique: true, Keys: []schema.Key{schema.Key{Column: colname, Desc: false}}})
+					conv.SrcSchema[tableName] = ctable
+				}
```

Comment:
This is non-trivial code, and deserves a comment on the line of -

```
\\ Convert unique column constraint in mysql to a corresponding unique index in Spanner since Spanner doesn't support unique constraints on columns.
```

Reply:
added comment